test/functorch/test_vmap.py

```diff
@@ -1178,6 +1178,7 @@ def f(x, y):
         self.assertEqual(out, f(None, y))
         self.assertEqual(out_dims, (None, None, None))
 
+    @xfailIfTorchDynamo
```

Comment:
What's going on here?

Reply:
This does not raise an error under torch.compile

Reply:
Let's ignore this for now, file an issue for it so we come back to it.